lib/improver/nbhood.py

```diff
@@ -170,6 +172,36 @@ def get_neighbourhood_width_in_grid_cells(
             )
         return grid_cells_x, grid_cells_y
 
+    @staticmethod
+    def adjust_nsize_for_ens(ens_factor, num_ens, width_in_km):
+        """
+        Adjust neighbourhood size according to ensemble size.
+
+        Parameters
+        ----------
+        ens_factor : float
+            Defined Factor with which to adjust the neighbour size
+            If ens_factor = 1.0 this essentially conservers ensemble
+            members if every grid square is considered to be the
+            equivalent of an ensemble member.
+        num_ens : float
+            Number of realizations / ensemble members.
+        width_in_km : float
+            radius or width appropriate for a single forecast in km.
+
+        Returns
+        -------
+        new_width : float
+            new neighbourhood radius (km).
+
+        """
+        if num_ens <= 1.0:
```

Comment:
In previous versions I wanted to distinguish between deterministic data (which I set num_ens=0.0) and cubes with one realization hence the <=1.0. However the difference was no longer required and Stephen requested that num_ens be reverted back to 1.0 for deterministic data. Unless this is causing a problem I will leave this as <=1.0 so that we have the option of reverting back to num_ens =0.0 in the future if we do need to distinguish deterministic data from other data.